pkg/controllers/node/termination/suite_test.go

```diff
@@ -944,29 +956,6 @@ var _ = Describe("Termination", func() {
 			Expect(ok).To(BeTrue())
 			Expect(lo.FromPtr(m.GetHistogram().SampleCount)).To(BeNumerically("==", 1))
 		})
-		It("should update the eviction queueDepth metric when reconciling pods", func() {
```

Comment:
Nice! This means we can just use the controller-runtime metric for this! How does this affect the existing metric?